lib/api/list.js

```diff
@@ -43,6 +43,18 @@ List.prototype = {
 
   },
 
+  removeLeadsFromList: function(listId, leadIds) {
+    if (!_.isArray(leadIds) && !_.isEmpty(leadIds)) {
+      var msg = 'input must be an array of lead ids';
+      log.error(msg);
+      return Promise.reject(msg);
+    }
+
+    var path = util.createPath('lists', listId, 'leads.json');
+    leadIds = [].concat(leadIds); // ensure it's an array
```

Comment:
The conditional check above `!_.isArray(leadIds)` will render this call useless.  I like the fact that the API accepts an array, so I think removing this conversion is more ideal and still safe.